src/library_manager/llext_manager.c

```diff
@@ -75,9 +75,9 @@ static int llext_manager_load_data_from_storage(const struct llext *ext,
 						size_t size, uint32_t flags)
 {
 	unsigned int i;
-	int ret = llext_manager_align_map(vma, size, SYS_MM_MEM_PERM_RW);
-	const elf_shdr_t *shdr;
```

Comment:
Removing 'shdr' here will break compilation on this commit. This change should probably be moved to the next commit.